include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -850,6 +850,33 @@ def TTIR_UnsqueezeOp : TTIR_DPSOp<"unsqueeze"> {
     let hasVerifier = 1;
 }
 
+def TTIR_ClampOp : TTIR_DPSOp<"clamp"> {
```

Comment:
Should we add some verifier for the op? For example, should we check that the output shape is the same as the input shape?

Reply:
We are getting mlir graphs from different front ends which already perform verification. I think, It will be redundant to have verifier for basic ops where we only do one-to-one mapping.

Reply:
good point. I'll add verifier to check basic requirements. thanks for the suggestion.

Reply:
I have added verifier module for clamp op. thanks for the suggestion.

Reply:
Thanks, appreciated!